android/src/main/java/com/inbrain/InBrainSurveysModule.java

```diff
@@ -230,20 +247,18 @@ public void nativeSurveysReceived(List<Survey> surveys) {
     // ***** SHOW NATIVE SURVEY ******
     // *******************************
     @ReactMethod
-    public void showNativeSurvey(final String id, final String searchId, final Promise promise) {
+    public void showNativeSurvey(final String id, final String searchId, final boolean showOffers, final Promise promise) {
```

Comment:
I would like to propose renaming `showOffers` to `offersEnabled` to make all SDKs more similar, of course if it possible. At the moment the offers are showing, but who knows what the "enabled" flag will mean in the future.